resources/data/dictionary/sum_definitions.json

```diff
@@ -0,0 +1,304 @@
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_ceiling_fan_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_clothes_dryer_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_clothes_washer_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_cooling_fans_pumps_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_cooling_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_dehumidifier_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_dishwasher_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_electric_vehicle_charging_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_freezer_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_generator_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_heating_fans_pumps_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_heating_heat_pump_backup_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_heating_heat_pump_backup_fans_pumps_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_heating_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_permanent_spa_heater_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_permanent_spa_pump_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_hot_water_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_hot_water_recirc_pump_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_hot_water_solar_thermal_pump_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_lighting_exterior_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_lighting_garage_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_lighting_interior_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_mech_vent_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_mech_vent_precooling_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_mech_vent_preheating_lb",
+        "report_simulation_output.emissions_co_2_e_<emissions_scenario_name>_electricity_net_lb",
```

Comment:
I don't know what this PR is supposed to be doing, but if all of these items are supposed to sum to the total, then this net value should not be here.